web/src/features/map-controls/LanguageSelector.tsx

```diff
@@ -22,12 +23,19 @@ export function LanguageSelector() {
   return (
     <MapOptionSelector
       trigger={
-        <MapButton
-          icon={<HiLanguage size={20} style={{ strokeWidth: '0.5' }} />}
-          tooltipText={__('tooltips.selectLanguage')}
-        />
+        isMobile ? (
+          <Button icon={<HiLanguage size={21} />} className="mb-0">
+            {__('tooltips.selectLanguage')}
+          </Button>
+        ) : (
+          <MapButton
+            icon={<HiLanguage size={20} style={{ strokeWidth: '0.5' }} />}
+            tooltipText={__('tooltips.selectLanguage')}
+          />
+        )
```

Comment:
This piece of code seems and the one below is almost identical, does it make sense to try and combine them?